input.tf

```diff
@@ -70,6 +70,12 @@ variable features {
   }
 }
 
+variable vpc {
```

Comment:
Should this bubble up to the `features` variable as well?

Reply:
Nah, VPC is a feature, flow-logs an option to the VPC, just like we do for everythign else.

```toml
feature = {
  vpc = true
  ci = true
}
vpc = { 
  flow_logs = true
}

ci = {
 project = "foo"
 git_repo = "https:/fff/"
}
```

Reply:
Ok works for me